spel/minimal-linux.pkr.hcl

```diff
@@ -923,6 +923,23 @@ build {
       "echo Killing processes locking /oldroot",
       "fuser -vmk /oldroot",
     ]
+    except = [
+      "amazon-ebs.minimal-centos-9stream-hvm",
+      "amazon-ebs.minimal-ol-9-hvm",
+      "amazon-ebs.minimal-rhel-9-hvm",
+    ]
```

Comment:
We actually have a similar issue with getting the disk to release on EL7. It's less persistent, more of a race condition, but the presentation is very similar. I think the only reason on EL8 we don't see the problem is due to different handling of the /boot partition.

That saying, I don't think we need to conditionally execute the new script. Just run it on all builders, and remove this provisioner stanza here that is trying to unlock /oldroot.

Reply:
Looks like the logic does nothing to address that and may actually introduce different problems for EL7. Updating the `only`/`exclude` stanzas to ensure that EL7-based builds get the old behavior while everything else gets the new behavior.